docs/r2/tensorboard_scalars_and_keras_basic.ipynb

```diff
@@ -0,0 +1,452 @@
+        "!pip install -q tf-nightly-2.0-preview\n",
+        "# Load the TensorBoard notebook extension.\n",
+        "%load_ext tensorboard.notebook"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "1qIKtOBrqc9Y",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "from __future__ import absolute_import\n",
+        "from __future__ import division\n",
+        "from __future__ import print_function\n",
+        "\n",
+        "from datetime import datetime\n",
+        "from packaging import version\n",
+        "\n",
+        "import tensorflow as tf\n",
+        "from tensorflow import keras\n",
+        "\n",
+        "import numpy as np\n",
+        "\n",
+        "print(\"TensorFlow version: \", tf.__version__)\n",
+        "assert version.parse(tf.__version__).release[0] >= 2, \\\n",
+        "    \"This notebook requires TensorFlow 2.0 or above.\""
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "6YDAoNCN3ZNS",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Set up data for a simple regression\n",
+        "\n",
+        "You're now going to use Keras to calculate a regression, i.e., find the best line of fit for a paired data set. (While using neural networks and gradient descent is [overkill for this kind of problem](https://https://stats.stackexchange.com/questions/160179/do-we-need-gradient-descent-to-find-the-coefficients-of-a-linear-regression-mode), it does make for a very easy to understand example.)\n",
+        "\n",
+        "You're going to use TensorBoard to observe how training and test **loss** change across epochs. Hopefully, you'll see training and test loss decrease over time and then remain steady.\n",
+        "\n",
+        "First, generate 1000 data points roughly along the line *y = 0.5x + 2*. Split these data points into training and test sets. Your hope is that the neural net learns this relationship."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "j-ryO6OxnQH_",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "data_size = 1000\n",
+        "# 80% of the data is for training.\n",
+        "train_pct = 0.8\n",
+        "\n",
+        "train_size = int(data_size * train_pct)\n",
+        "\n",
+        "# Create some input data between -1 and 1 and randomize it.\n",
+        "X = np.linspace(-1, 1, data_size)\n",
+        "np.random.shuffle(X)\n",
+        "\n",
+        "# Generate the output data.\n",
+        "# f(X) = 0.5X + 2 + noise\n",
+        "Y = 0.5 * X + 2 + np.random.normal(0, 0.05, (data_size, ))\n",
+        "\n",
+        "# Split into test and train pairs.\n",
+        "X_train, Y_train = X[:train_size], Y[:train_size]\n",
```

Comment:
Should we standardize on x_train / y_train (instead of X_train / Y_train)? That's what I've seen in the other docs.

Reply:
Fixed.